File: Vasavi_National_Level_Contest/Testing_and_Miscellaneous/integer_limit_test.java
class Main
{
    public static void main(String[] args) {
        long fact[]= new long[100];
        fact[0]=1;
        for(int i=1;i<100;i++){
            fact[i]=fact[i-1]*i;
            if(fact[i]<fact[i-1])
                System.out.println("error"+i);
            System.out.println(i+" "+fact[i]);
        }
    }
}

File: Vasavi_National_Level_Contest/Testing_and_Miscellaneous/integer_limit_test.c
#include<stdio.h>
#include<stdlib.h>
#include<string.h>
#include<limits.h>
#include<ctype.h>
#include<math.h>
#include<stdbool.h>
#include<assert.h>
typedef unsigned long long int ull;
typedef long long int ll;
int main()
{
    ll fact[100]={1};
    for(int i=1;i<100;i++)
    {
        fact[i]=fact[i-1]*i;
        if(fact[i]<fact[i-1])
            printf("Error: %d\n",i);
        printf("%d %lld\n",i,fact[i]);
        
    }
    return 0;
}